spec/support/coronavirus_feature_steps.rb

```diff
@@ -356,6 +356,21 @@ def then_the_timeline_entry_is_updated
   expect(@timeline_entry.reload.content).to eq("##Form content")
 end
 
+# Deleting timeline entries
+
+def when_i_delete_a_timeline_entry
+  set_up_github_data(@coronavirus_page)
+
+  page.accept_alert "Are you sure?" do
+    page.find("a[href=\"/coronavirus/landing/timeline_entries/#{@timeline_entry.id}\"]", text: "Delete").click
+  end
+end
+
+def then_i_can_see_the_timeline_entry_has_been_deleted
+  expect(page).to have_text("Timeline entry was successfully deleted.")
+  expect(page).not_to(have_text(@timeline_entry.heading))
```

Comment:
```suggestion
  expect(page).not_to have_text(@timeline_entry.heading)
```

Just minor slip out of RSpec conventions